Dockerfile_build_ubuntu64_git

```diff
@@ -27,7 +27,7 @@ VOLUME $PROJECT_DIR
 
 
 # Install go
-ENV GO_VERSION 1.13
+ENV GO_VERSION 1.15.10
```

Comment:
Do we still use this Dockerfile? I thought we deleted the go_next tests

Reply:
Will delete